### PR TITLE
feat(attendance): add work from home as a status option

### PR DESCRIPTION
**Pull request by DeeMysterio**
**Task:** [TASK-2019-00866](https://digithinkit.global/desk#Form/Task/TASK-2019-00866)

**Additions:**

- Added a "Work From Home" option in status field.

- Provided a button saying "Mark Work From Home" on the Employee Attendance Tool.

- Changed the styling of the buttons on Employee Attendance Tool and indicators on the attendance list view.

**Screenshots/Gifs:**

Screenshot 2019-12-23 at 3 47 58 PM

Screenshot 2019-12-23 at 3 53 46 PM

### CODE REVIEW

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           

Complexity increasing per file
==============================
- erpnext/hr/doctype/attendance/attendance_list.js  4
         

Clones added
============
- erpnext/hr/doctype/employee_attendance_tool/employee_attendance_tool.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/DigiThinkIT/erpnext/pullRequest?prid=4723003&bid=15768291)

File: erpnext/hr/doctype/attendance/attendance_list.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 frappe.listview_settings['Attendance'] = {
 	add_fields: ["status", "attendance_date"],
-	get_indicator: function(doc) {
-		return [__(doc.status), doc.status=="Present" ? "green" : "darkgrey", "status,=," + doc.status];
+	get_indicator: function (doc) {
+		if (doc.status == "Present" || doc.status == "Work From Home") {
```
Comment (Alchez) [inline erpnext/hr/doctype/attendance/attendance_list.js]: Combine the conditions, like:

```javascript
if (["Present", "Work From Home"].includes(doc.status) {
	// do things
}
```

Which is similar to doing the `<value> in <iterable>` check in Python

**Review by Alchez — CHANGES_REQUESTED** [review]
Also fix Codacy errors

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- erpnext/hr/doctype/attendance/attendance_list.js  3
         

Clones added
============
- erpnext/hr/doctype/employee_attendance_tool/employee_attendance_tool.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/DigiThinkIT/erpnext/pullRequest?prid=4723003&bid=15768291)

File: erpnext/hr/doctype/attendance/attendance_list.js [hunk 2]
```diff
@@ -1,6 +1,12 @@
 frappe.listview_settings['Attendance'] = {
 	add_fields: ["status", "attendance_date"],
-	get_indicator: function(doc) {
-		return [__(doc.status), doc.status=="Present" ? "green" : "darkgrey", "status,=," + doc.status];
+	get_indicator: function (doc) {
+		if (["Present", "Work From Home"].includes(doc.status)) {
+			return [__(doc.status), "green", "status,=," + doc.status];
+		} else if (doc.status == "Absent" || doc.status == "On Leave") {
```
Comment (Alchez) [inline erpnext/hr/doctype/attendance/attendance_list.js]: Here too

**Review by Alchez — CHANGES_REQUESTED** [review]

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- erpnext/hr/doctype/attendance/attendance_list.js  2
         

Clones added
============
- erpnext/hr/doctype/employee_attendance_tool/employee_attendance_tool.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/DigiThinkIT/erpnext/pullRequest?prid=4723003&bid=15768291)

**Review by DeveloperMetal — APPROVED** [review]

**Comment by Alchez** [comment]
@DeeMysterio, make this upstream and `bloomstack-v12-staging` as well

**Pull request merged**